docs/DeveloperGuide.md

```diff
@@ -432,7 +432,16 @@ _{more aspects and alternatives to be added}_
 - Modify the format of `DATE` to be `d-M-yyyy` and `DATETIME` to be `d-M-yyyy`T`H:mm` instead.
 - This allows FINDvisor to accept both single and double-digits day, month and hour values as valid `DATE` and `DATETIME` values and would not require users to pad these single digit values with a leading zero.
 
-### 3. Show warning to user when scheduling an overlapping meeting
+### 5. Specify error message for reschedule and schedule commands
+
+**Current Implementation**:
+- When users input a invalid date for any parameters, the error message states that it is an invalid command format, although the issue is that an invalid date is passed.
+
+**Proposed Implementation**:
+- The error message should specify which of the given parameters are failing instead of prompting an invalid command format.
+- This can be applied for both reschedule and schedule as they go through the same checks.
```

Comment:
I think the phrasing of the proposed implementation can be improved? @Dethada @Javiery3889 @Arixeyeion Do you have any suggestions for improvement?

Reply:
For line 441
```suggestion
- The error message should mention which parameter failed and the reason for the failure, e.g. `The DateTime 31-02-2024T12:00 is invalid for START_DATETIME parameter. Please use the following format: dd-MM-yyyy'T'HH:mm, e.g. 02-02-2024T22:00.`
```

Reply:
I think the phrasing is good, but example can be improved? Perhaps a shorter message would be better such as `START_DATETIME is invalid`?

Do we need multiple error messages, where one is for incorrect `DATETIME` format and another is for incorrect `DATETIME` value about a particular `DATETIME`?

Reply:
I think better to have both.

Reply:
@KohGuanZeh @YuZizhen 
I would suggest:
- The error message should mention which parameter failed and the reason for the failure, e.g. `The DateTime 31-02-2024T12:00 is invalid. Please use the following format: dd-MM-yyyy'T'HH:mm, e.g. 02-02-2024T22:00.`

Because the command is left in the Commnd Box when an error occurs, so it is sufficient for the user input to be displayed back to the user for the DateTime that cannot be parsed.